docs/CONTRIBUTING.md

```diff
@@ -60,39 +63,53 @@ To set up your development environment, please follow these steps:
    git clone https://github.com/clerk/javascript
    ```
 
-2. Install the dependencies. We're using npm workspaces, so you **should always run `npm install` from the root of the monorepo**, as it will install dependencies for all the packages:
+1. Install the dependencies. We're using npm workspaces, so you **should always run `npm install` from the root of the monorepo**, as it will install dependencies for all the packages:
```

Comment:
❓ shouldn't this be 2? The 1st item is the clone repo step above.

Reply:
You can use `1.` for everything, then you don't have to manually number things and change numbers if you add items in between: https://www.markdownguide.org/basic-syntax/#ordered-lists